Tests can't be generated in the debug IDE due to

**Description**

No tests are generated in the debug IDE (`runIDE` Gradle targe of `utbot-intellij`). Generation fails, and the exception is thrown and caught by the debug IDEA.

It seems that `log4j2` configuration file can't be found (see below for the log).

**To Reproduce**

Start the debug IDEA (`runIDE` run configuration / Gradle target) and invoke the plugin to generate any test suite.
A sample code:
```
package sample;

public class SimpleExample {
    public int bar(int a) {
        return a + 1;
    }
}
```

Try to generate a test for `bar` method or for the entire class.

The bug is reproducing using the current `main` branch (last commit: `56bf6ab79fab2631a52a1b86da249c60c36ae73e`),

**Expected behavior**

A test suite should be generated.

**Actual behavior**

The plugin fails with an exception that is caught and displayed by the debug IDE.
```
kotlinx.coroutines.JobCancellationException: Job was cancelled; job=CompletableDeferredImpl{Cancelled}@67026c6b
	at kotlinx.coroutines.JobSupport.cancel(JobSupport.kt:1578)
	at kotlinx.coroutines.Job$DefaultImpls.cancel$default(Job.kt:183)
	at com.jetbrains.rd.framework.util.LifetimeCoroutineUtilKt$synchronizeWith$1.invoke(LifetimeCoroutineUtil.kt:15)
	at com.jetbrains.rd.framework.util.LifetimeCoroutineUtilKt$synchronizeWith$1.invoke(LifetimeCoroutineUtil.kt:15)
	at com.jetbrains.rd.util.lifetime.LifetimeDefinition.destruct(RLifetime.kt:427)
	at com.jetbrains.rd.util.lifetime.LifetimeDefinition.terminate(RLifetime.kt:408)
	at com.jetbrains.rd.util.lifetime.LifetimeDefinition.destruct(RLifetime.kt:431)
	at com.jetbrains.rd.util.lifetime.LifetimeDefinition.terminate(RLifetime.kt:408)
	at com.jetbrains.rd.util.lifetime.LifetimeDefinition.terminate$default(RLifetime.kt:375)
	at org.utbot.rd.LifetimedProcessIml$2.invokeSuspend(LifetimedProcess.kt:80)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
	at com.jetbrains.rd.framework.util.SchedulerCoroutineDispatcher$dispatch$1.invoke(ISchedulerCoroutineUtil.kt:8)
	at com.jetbrains.rd.framework.util.SchedulerCoroutineDispatcher$dispatch$1.invoke(ISchedulerCoroutineUtil.kt:8)
	at com.jetbrains.rd.util.threading.SingleThreadSchedulerBase.queue$lambda-3(SingleThreadScheduler.kt:41)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
```

The log from the "main" IDEA:
```
java - /home/dtim/.jdks/corretto-11.0.15/bin/java
classpath - "/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/utbot-intellij-2022.10-SNAPSHOT.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/kryo-5.0.0-RC1.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/guava-31.1-android.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/txw2-2.4.0-b180830.0438.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/utbot-rd-2022.10-SNAPSHOT.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/utbot-api-2022.10-SNAPSHOT.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/jaxb-api-2.4.0-b180830.0359.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/utbot-core-2022.10-SNAPSHOT.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/javasmt-solver-z3-4.8.9-sosy1.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/utbot-fuzzers-2022.10-SNAPSHOT.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/utbot-summary-2022.10-SNAPSHOT.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/jaxb-runtime-2.4.0-b180830.0438.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/utbot-framework-2022.10-SNAPSHOT.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/utbot-framework-api-2022.10-SNAPSHOT.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/utbot-instrumentation-2022.10-SNAPSHOT.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/listenablefuture-9999.0-empty-to-avoid-conflict-with-guava.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/asm-9.2.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/jna-5.5.0.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/gson-2.8.9.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/rgxgen-1.3.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/axml-2.0.0.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/heros-1.2.2.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/stax-ex-1.8.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/kryo5-5.3.0.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/jsr305-3.0.2.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/jasmin-3.0.2.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/parso-2.0.12.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/asm-util-9.2.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/asm-tree-9.2.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/minlog-1.3.0.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/dexlib2-2.5.2.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/objenesis-3.2.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/polyglot-2006.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/soot-3adf23c3.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/smile-io-2.6.0.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/commons-io-2.7.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/java_cup-0.9.2.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/httpcore5-5.1.1.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/asm-commons-9.2.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/httpclient5-5.1.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/commons-csv-1.8.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/smile-nlp-2.6.0.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/asm-analysis-9.2.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/smile-core-2.6.0.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/smile-data-2.6.0.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/slf4j-api-1.7.32.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/commons-text-1.9.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/annotations-13.0.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/opentest4j-1.2.0.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/smile-math-2.6.0.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/rd-core-2022.3.1.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/log4j-api-2.13.3.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/smile-graph-2.6.0.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/log4j-core-2.13.3.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/byte-buddy-1.12.4.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/reflectasm-1.11.7.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/mockito-core-4.2.0.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/FastInfoset-1.2.15.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/commons-codec-1.15.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/httpcore5-h2-5.1.1.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/smile-kotlin-2.6.0.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/commons-lang3-3.11.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/functionaljava-4.2.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/jna-platform-5.5.0.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/failureaccess-1.0.1.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/jackson-core-2.12.3.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/commons-logging-1.2.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/checker-qual-3.12.0.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/mockito-inline-4.2.0.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/kotlin-stdlib-1.7.20.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/protobuf-java-3.21.2.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/kotlin-logging-1.8.3.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/org.jacoco.core-0.8.8.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/kryo-serializers-0.45.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/rd-framework-2022.3.1.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/kotlin-reflect-1.7.20.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/javaparser-core-3.22.1.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/j2objc-annotations-1.3.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/jackson-databind-2.12.3.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/junit-jupiter-api-5.8.1.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/log4j-slf4j-impl-2.13.3.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/org.jacoco.report-0.8.8.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/xmlpull-1.1.3.4d_b4_min.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/byte-buddy-agent-1.12.4.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/kotlin-stdlib-jdk7-1.7.20.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/protobuf-java-util-3.21.2.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/kotlin-stdlib-jdk8-1.7.20.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/javax.activation-api-1.2.0.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/javax.annotation-api-1.3.2.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/junit-jupiter-engine-5.8.1.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/jackson-annotations-2.12.3.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/junit-jupiter-params-5.8.1.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/junit-platform-engine-1.8.1.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/kotlin-stdlib-common-1.7.20.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/junit-platform-commons-1.8.1.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/istack-commons-runtime-3.0.7.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/jackson-module-kotlin-2.12.3.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/error_prone_annotations-2.11.0.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/kotlinx-coroutines-core-jvm-1.6.3.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/kotlinx-collections-immutable-jvm-0.3.4.jar"
port - 38635
java - /home/dtim/.jdks/corretto-11.0.15/bin/java
classpath - "/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/utbot-intellij-2022.10-SNAPSHOT.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/kryo-5.0.0-RC1.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/guava-31.1-android.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/txw2-2.4.0-b180830.0438.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/utbot-rd-2022.10-SNAPSHOT.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/utbot-api-2022.10-SNAPSHOT.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/jaxb-api-2.4.0-b180830.0359.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/utbot-core-2022.10-SNAPSHOT.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/javasmt-solver-z3-4.8.9-sosy1.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/utbot-fuzzers-2022.10-SNAPSHOT.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/utbot-summary-2022.10-SNAPSHOT.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/jaxb-runtime-2.4.0-b180830.0438.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/utbot-framework-2022.10-SNAPSHOT.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/utbot-framework-api-2022.10-SNAPSHOT.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/utbot-instrumentation-2022.10-SNAPSHOT.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/listenablefuture-9999.0-empty-to-avoid-conflict-with-guava.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/asm-9.2.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/jna-5.5.0.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/gson-2.8.9.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/rgxgen-1.3.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/axml-2.0.0.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/heros-1.2.2.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/stax-ex-1.8.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/kryo5-5.3.0.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/jsr305-3.0.2.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/jasmin-3.0.2.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/parso-2.0.12.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/asm-util-9.2.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/asm-tree-9.2.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/minlog-1.3.0.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/dexlib2-2.5.2.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/objenesis-3.2.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/polyglot-2006.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/soot-3adf23c3.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/smile-io-2.6.0.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/commons-io-2.7.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/java_cup-0.9.2.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/httpcore5-5.1.1.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/asm-commons-9.2.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/httpclient5-5.1.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/commons-csv-1.8.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/smile-nlp-2.6.0.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/asm-analysis-9.2.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/smile-core-2.6.0.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/smile-data-2.6.0.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/slf4j-api-1.7.32.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/commons-text-1.9.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/annotations-13.0.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/opentest4j-1.2.0.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/smile-math-2.6.0.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/rd-core-2022.3.1.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/log4j-api-2.13.3.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/smile-graph-2.6.0.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/log4j-core-2.13.3.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/byte-buddy-1.12.4.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/reflectasm-1.11.7.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/mockito-core-4.2.0.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/FastInfoset-1.2.15.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/commons-codec-1.15.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/httpcore5-h2-5.1.1.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/smile-kotlin-2.6.0.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/commons-lang3-3.11.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/functionaljava-4.2.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/jna-platform-5.5.0.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/failureaccess-1.0.1.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/jackson-core-2.12.3.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/commons-logging-1.2.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/checker-qual-3.12.0.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/mockito-inline-4.2.0.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/kotlin-stdlib-1.7.20.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/protobuf-java-3.21.2.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/kotlin-logging-1.8.3.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/org.jacoco.core-0.8.8.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/kryo-serializers-0.45.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/rd-framework-2022.3.1.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/kotlin-reflect-1.7.20.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/javaparser-core-3.22.1.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/j2objc-annotations-1.3.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/jackson-databind-2.12.3.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/junit-jupiter-api-5.8.1.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/log4j-slf4j-impl-2.13.3.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/org.jacoco.report-0.8.8.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/xmlpull-1.1.3.4d_b4_min.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/byte-buddy-agent-1.12.4.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/kotlin-stdlib-jdk7-1.7.20.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/protobuf-java-util-3.21.2.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/kotlin-stdlib-jdk8-1.7.20.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/javax.activation-api-1.2.0.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/javax.annotation-api-1.3.2.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/junit-jupiter-engine-5.8.1.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/jackson-annotations-2.12.3.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/junit-jupiter-params-5.8.1.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/junit-platform-engine-1.8.1.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/kotlin-stdlib-common-1.7.20.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/junit-platform-commons-1.8.1.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/istack-commons-runtime-3.0.7.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/jackson-module-kotlin-2.12.3.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/error_prone_annotations-2.11.0.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/kotlinx-coroutines-core-jvm-1.6.3.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/kotlinx-collections-immutable-jvm-0.3.4.jar"
port - 38635
java - /home/dtim/.jdks/corretto-11.0.15/bin/java
classpath - "/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/utbot-intellij-2022.10-SNAPSHOT.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/kryo-5.0.0-RC1.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/guava-31.1-android.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/txw2-2.4.0-b180830.0438.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/utbot-rd-2022.10-SNAPSHOT.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/utbot-api-2022.10-SNAPSHOT.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/jaxb-api-2.4.0-b180830.0359.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/utbot-core-2022.10-SNAPSHOT.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/javasmt-solver-z3-4.8.9-sosy1.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/utbot-fuzzers-2022.10-SNAPSHOT.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/utbot-summary-2022.10-SNAPSHOT.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/jaxb-runtime-2.4.0-b180830.0438.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/utbot-framework-2022.10-SNAPSHOT.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/utbot-framework-api-2022.10-SNAPSHOT.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/utbot-instrumentation-2022.10-SNAPSHOT.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/listenablefuture-9999.0-empty-to-avoid-conflict-with-guava.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/asm-9.2.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/jna-5.5.0.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/gson-2.8.9.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/rgxgen-1.3.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/axml-2.0.0.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/heros-1.2.2.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/stax-ex-1.8.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/kryo5-5.3.0.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/jsr305-3.0.2.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/jasmin-3.0.2.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/parso-2.0.12.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/asm-util-9.2.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/asm-tree-9.2.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/minlog-1.3.0.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/dexlib2-2.5.2.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/objenesis-3.2.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/polyglot-2006.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/soot-3adf23c3.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/smile-io-2.6.0.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/commons-io-2.7.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/java_cup-0.9.2.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/httpcore5-5.1.1.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/asm-commons-9.2.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/httpclient5-5.1.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/commons-csv-1.8.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/smile-nlp-2.6.0.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/asm-analysis-9.2.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/smile-core-2.6.0.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/smile-data-2.6.0.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/slf4j-api-1.7.32.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/commons-text-1.9.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/annotations-13.0.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/opentest4j-1.2.0.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/smile-math-2.6.0.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/rd-core-2022.3.1.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/log4j-api-2.13.3.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/smile-graph-2.6.0.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/log4j-core-2.13.3.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/byte-buddy-1.12.4.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/reflectasm-1.11.7.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/mockito-core-4.2.0.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/FastInfoset-1.2.15.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/commons-codec-1.15.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/httpcore5-h2-5.1.1.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/smile-kotlin-2.6.0.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/commons-lang3-3.11.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/functionaljava-4.2.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/jna-platform-5.5.0.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/failureaccess-1.0.1.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/jackson-core-2.12.3.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/commons-logging-1.2.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/checker-qual-3.12.0.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/mockito-inline-4.2.0.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/kotlin-stdlib-1.7.20.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/protobuf-java-3.21.2.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/kotlin-logging-1.8.3.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/org.jacoco.core-0.8.8.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/kryo-serializers-0.45.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/rd-framework-2022.3.1.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/kotlin-reflect-1.7.20.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/javaparser-core-3.22.1.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/j2objc-annotations-1.3.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/jackson-databind-2.12.3.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/junit-jupiter-api-5.8.1.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/log4j-slf4j-impl-2.13.3.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/org.jacoco.report-0.8.8.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/xmlpull-1.1.3.4d_b4_min.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/byte-buddy-agent-1.12.4.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/kotlin-stdlib-jdk7-1.7.20.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/protobuf-java-util-3.21.2.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/kotlin-stdlib-jdk8-1.7.20.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/javax.activation-api-1.2.0.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/javax.annotation-api-1.3.2.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/junit-jupiter-engine-5.8.1.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/jackson-annotations-2.12.3.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/junit-jupiter-params-5.8.1.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/junit-platform-engine-1.8.1.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/kotlin-stdlib-common-1.7.20.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/junit-platform-commons-1.8.1.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/istack-commons-runtime-3.0.7.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/jackson-module-kotlin-2.12.3.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/error_prone_annotations-2.11.0.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/kotlinx-coroutines-core-jvm-1.6.3.jar;/home/dtim/Projects/UTBotJava/utbot-intellij/build/idea-sandbox/plugins/utbot-intellij/lib/kotlinx-collections-immutable-jvm-0.3.4.jar"
port - 38635
logFile - /tmp/UTBot/rdEngineProcessLogs/109989167155163723-0.log
logFile - /tmp/UTBot/rdEngineProcessLogs/109989167155163723-0.log
logFile - /tmp/UTBot/rdEngineProcessLogs/109989167155163723-0.log
Debug | ByteBufferAsyncProcessor  | 232:ApplicationImpl pooled thread 28 @coroutine#7 | PAUSE ('Socket not connected') :: {id = ServerSocket/Sender, state = 'Initialized'} 
Debug | Server                    | 251:ServerSocket | ServerSocket: listening /127.0.0.1:38635 
Info  | Server                    | 132:UtCoroutineScheduler @coroutine#8 | ServerSocket: start terminating lifetime 
Debug | Server                    | 132:UtCoroutineScheduler @coroutine#8 | ServerSocket: shutting down ack sending executor 
Debug | Server                    | 132:UtCoroutineScheduler @coroutine#8 | ServerSocket: send buffer stopped, success: true 
Debug | Server                    | 132:UtCoroutineScheduler @coroutine#8 | ServerSocket: closing socket 
Error | Lifetime                  | 232:ApplicationImpl pooled thread 28 | Lifetime `Anonymous` [Terminating, executing=0, resources=3]: exception on termination of resource: () -> kotlin.Unit | java.util.concurrent.CancellationException
	at com.jetbrains.rd.util.lifetime.RLifetimeKt.throwIfNotAlive(RLifetime.kt:523)
	at org.utbot.intellij.plugin.process.EngineProcess.engineModel(EngineProcess.kt:96)
	at org.utbot.intellij.plugin.process.EngineProcess.access$engineModel(EngineProcess.kt:54)
	at org.utbot.intellij.plugin.process.EngineProcess$forceTermination$1.invokeSuspend(EngineProcess.kt:296)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
	at kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:284)
	at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:85)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:59)
	at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Builders.kt:38)
	at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source)
	at org.utbot.intellij.plugin.process.EngineProcess.forceTermination(EngineProcess.kt:294)
	at org.utbot.intellij.plugin.process.EngineProcess$1.invoke(EngineProcess.kt:370)
	at org.utbot.intellij.plugin.process.EngineProcess$1.invoke(EngineProcess.kt:369)
	at com.jetbrains.rd.util.lifetime.LifetimeDefinition.destruct(RLifetime.kt:427)
	at com.jetbrains.rd.util.lifetime.LifetimeDefinition.terminate(RLifetime.kt:408)
	at com.jetbrains.rd.util.lifetime.LifetimeDefinition.destruct(RLifetime.kt:431)
	at com.jetbrains.rd.util.lifetime.LifetimeDefinition.terminate(RLifetime.kt:408)
	at com.jetbrains.rd.util.lifetime.LifetimeDefinition.terminate$default(RLifetime.kt:375)
	at org.utbot.intellij.plugin.generator.UtTestsDialogProcessor$createTests$1$1.run(UtTestsDialogProcessor.kt:348)
	at com.intellij.openapi.progress.impl.CoreProgressManager.startTask(CoreProgressManager.java:442)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.startTask(ProgressManagerImpl.java:114)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcessWithProgressAsynchronously$5(CoreProgressManager.java:493)
	at com.intellij.openapi.progress.impl.ProgressRunner.lambda$submit$3(ProgressRunner.java:252)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:188)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$executeProcessUnderProgress$12(CoreProgressManager.java:608)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:683)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computeUnderProgress(CoreProgressManager.java:639)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:607)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:60)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:175)
	at com.intellij.openapi.progress.impl.ProgressRunner.lambda$submit$4(ProgressRunner.java:252)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:702)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:699)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1.run(Executors.java:699)
	at java.base/java.lang.Thread.run(Thread.java:833)

2022-10-07 18:31:22,318 [ 556683] SEVERE - #c.i.o.p.Task - Job was cancelled
kotlinx.coroutines.JobCancellationException: Job was cancelled; job=CompletableDeferredImpl{Cancelled}@4b9f06ac
	at kotlinx.coroutines.JobSupport.cancel(JobSupport.kt:1578)
	at kotlinx.coroutines.Job$DefaultImpls.cancel$default(Job.kt:183)
	at com.jetbrains.rd.framework.util.LifetimeCoroutineUtilKt$synchronizeWith$1.invoke(LifetimeCoroutineUtil.kt:15)
	at com.jetbrains.rd.framework.util.LifetimeCoroutineUtilKt$synchronizeWith$1.invoke(LifetimeCoroutineUtil.kt:15)
	at com.jetbrains.rd.util.lifetime.LifetimeDefinition.destruct(RLifetime.kt:427)
	at com.jetbrains.rd.util.lifetime.LifetimeDefinition.terminate(RLifetime.kt:408)
	at com.jetbrains.rd.util.lifetime.LifetimeDefinition.destruct(RLifetime.kt:431)
	at com.jetbrains.rd.util.lifetime.LifetimeDefinition.terminate(RLifetime.kt:408)
	at com.jetbrains.rd.util.lifetime.LifetimeDefinition.terminate$default(RLifetime.kt:375)
	at org.utbot.rd.LifetimedProcessIml$2.invokeSuspend(LifetimedProcess.kt:80)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
	at com.jetbrains.rd.framework.util.SchedulerCoroutineDispatcher$dispatch$1.invoke(ISchedulerCoroutineUtil.kt:8)
	at com.jetbrains.rd.framework.util.SchedulerCoroutineDispatcher$dispatch$1.invoke(ISchedulerCoroutineUtil.kt:8)
	at com.jetbrains.rd.util.threading.SingleThreadSchedulerBase.queue$lambda-3(SingleThreadScheduler.kt:41)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
2022-10-07 18:31:22,319 [ 556684] SEVERE - #c.i.o.p.Task - IntelliJ IDEA 2022.2.2  Build #IC-222.4167.29
2022-10-07 18:31:22,319 [ 556684] SEVERE - #c.i.o.p.Task - JDK: 17.0.4; VM: OpenJDK 64-Bit Server VM; Vendor: JetBrains s.r.o.
2022-10-07 18:31:22,319 [ 556684] SEVERE - #c.i.o.p.Task - OS: Linux
2022-10-07 18:31:22,319 [ 556684] SEVERE - #c.i.o.p.Task - Plugin to blame: UnitTestBot version: 2022.10-SNAPSHOT
Info  | Server                    | 132:UtCoroutineScheduler @coroutine#8 | ServerSocket: termination finished 
Debug | Server                    | 251:ServerSocket | ServerSocket closed with exception: java.net.SocketException: Socket closed 
Debug | Server                    | 251:ServerSocket | ServerSocket: waiting for reconnection on /127.0.0.1:38635. 
Debug | Server                    | 251:ServerSocket | ServerSocket: terminated. 

```

Contents of `/tmp/UTBot/rdEngineProcessLogs/109989167155163723-0.log`:
```
Error: Could not find or load main class "-Dlog4j2.configurationFile=.tmp.UTBot.rdEngineProcessLogConfigurations.epl15304543813638643830.xml"
Caused by: java.lang.ClassNotFoundException: "-Dlog4j2.configurationFile=.tmp.UTBot.rdEngineProcessLogConfigurations.epl15304543813638643830.xml"
/tmp/U
```

**Environment**

I can reproduce this behavior using IDEA Ultimate 2022.2.2 as the development IDE and IDEA Community 2022.2.2 as the debug IDE under Ubuntu 20.04.5 LTS. It seems that test generation settings do not matter here.

--- a/utbot-intellij/src/main/kotlin/org/utbot/intellij/plugin/process/EngineProcess.kt
+++ b/utbot-intellij/src/main/kotlin/org/utbot/intellij/plugin/process/EngineProcess.kt
@@ -11,10 +11,7 @@ import kotlinx.coroutines.runBlocking
 import kotlinx.coroutines.sync.Mutex
 import kotlinx.coroutines.sync.withLock
 import mu.KotlinLogging
-import org.utbot.common.AbstractSettings
-import org.utbot.common.getPid
-import org.utbot.common.osSpecificJavaExecutable
-import org.utbot.common.utBotTempDirectory
+import org.utbot.common.*
 import org.utbot.framework.UtSettings
 import org.utbot.framework.codegen.*
 import org.utbot.framework.codegen.model.UtilClassKind
@@ -81,8 +78,9 @@ class EngineProcess(parent: Lifetime, val project: Project) {
                 )
             }.toPath()
             realPath = configPath
+            logger.info("log configuration path - ${realPath!!.pathString}")
         }
-        return realPath!!.pathString
+        return realPath.pathString
     }
 
     private fun debugArgument(): String {
@@ -104,14 +102,14 @@ class EngineProcess(parent: Lifetime, val project: Project) {
                     val java =
                         JdkInfoService.jdkInfoProvider.info.path.resolve("bin${File.separatorChar}${osSpecificJavaExecutable()}").toString()
                     val cp = (this.javaClass.classLoader as PluginClassLoader).classPath.baseUrls.joinToString(
-                        separator = ";",
+                        separator = if (isWindows) ";" else ":",
                         prefix = "\"",
                         postfix = "\""
                     )
                     val classname = "org.utbot.framework.process.EngineMainKt"
                     val javaVersionSpecificArguments = OpenModulesContainer.javaVersionSpecificArguments
                     val directory = WorkingDirService.provide().toFile()
-                    val log4j2ConfigFile = "\"-Dlog4j2.configurationFile=${getOrCreateLogConfig()}\""
+                    val log4j2ConfigFile = "-Dlog4j2.configurationFile=${getOrCreateLogConfig()}"
                     val debugArg = debugArgument()
                     logger.info { "java - $java\nclasspath - $cp\nport - $port" }
                     val cmd = mutableListOf<String>(java, "-ea")

